tests.py

```diff
@@ -1,51 +1,128 @@
-from crawler import validated, validated_link, page_properties, map_page, map_site
-from urllib.parse import urlparse
+from crawler import (
+    is_validated,
+    make_url_absolute,
+    filtered_out_links,
+    updated_urls_list,
+    page_title,
+    page_links,
+    site_map,
+)
 
 
+def test_is_validated0():
```

Comment:
You need to describe what these test are actually testing. You can either change the name of the function like `test_is_validated_full_domain_with_protocol` or add a docstring
`""" Test case for a full domain. """`

This applies to all your test cases.